.travis.yml

```diff
@@ -46,6 +51,6 @@ deploy:
     file_glob: true
     file: "netdata*.tar.*"
     on:
-      condition: $CC = gcc
+      condition: $CC = gcc && $TRAVIS_OS_NAME
```

Comment:
hm.. I think this comment is for @philwhineray .
@philwhineray could you please check it?